rotkehlchen/chain/ethereum/adex/typing.py

```diff
@@ -55,28 +55,38 @@ def __str__(self) -> str:
 
 
 @dataclass(init=True, repr=True)
-class Bond:
-    tx_hash: HexStr  # from bond.id
+class AdexEvent:
```

Comment:
@LefterisJP this is the common event, subclassed for Bond, Unbond, UnbondRequest and ChannelWithdraw. Is there anything else I should take into account for the PnL report? Let me know